components/temporarycomponents/YoutubeLinkBanner.tsx

```diff
@@ -0,0 +1,90 @@
+import { useTranslation } from 'next-i18next';
+import { getCountryVariant } from 'utils/locales';
+import YoutubeLinkIcon from 'public/icons/youtube-link.svg';
+import Image from 'next/image';
+import { ReactNode } from 'react';
+import Lekarna_UK_mobile from 'public/youtubebanner/lekarna-uk--mobile.png';
+import Lekarna_CZ_mobile from 'public/youtubebanner/lekarna-cz--mobile.png';
+import Lekarna_UK_desktop from 'public/youtubebanner/lekarna-uk--desktop.png';
+import Lekarna_CZ_desktop from 'public/youtubebanner/lekarna-cz--desktop.png';
+import Person_UK from 'public/youtubebanner/person-uk.png';
+import Person_CZ from 'public/youtubebanner/person-cz.png';
+
+const IMAGES: Record<string, ReactNode> = {
+  'lekarna-uk-mobile': <Image src={Lekarna_UK_mobile} alt="Ikona se slovem aптека" />,
+  'lekarna-cz-mobile': <Image src={Lekarna_CZ_mobile} alt="Ikona se slovem lekárna" />,
+  'lekarna-uk-desktop': <Image src={Lekarna_UK_desktop} alt="Ikona se slovem aптека" />,
+  'lekarna-cz-desktop': <Image src={Lekarna_CZ_desktop} alt="Ikona se slovem lekárna" />,
+};
+
+const YTVIDEOCODECZ: Array<string> = [
```

Comment:
Just a stylistic nitpick, feel free to use `_` to make constants more readable `YOUTUBE_IDS_CZ` :)